src/nplinker/scoring/link_graph.py

```diff
@@ -76,17 +78,17 @@ def __init__(self) -> None:
 
             Display the empty LinkGraph object:
             >>> lg
-            |    |   Object 1 |   Object 2 |   Metcalf Score |   Rosetta Score |
-            |----|------------|------------|-----------------|-----------------|
+            |    | Genomic Object Type   | Genomic Object ID   | Metabolomic Object Type   | Metabolomic Object ID   | Metcalf Score   | Rosetta Score   |
```

Comment:
The docstring line length should be ≤100, same for other type of lines. But this docstring can be an exception for clarity.